intercept/src/controls.h

```diff
@@ -172,6 +172,7 @@ CLI_CONTROL( cl_uint,       AubCaptureStartWait,                    0,      "The
 CLI_CONTROL( cl_uint,       AubCaptureEndWait,                      0,      "The Intercept Layer for OpenCL Applications will wait for this many milliseconds before ending aub capture.")
 
 CLI_CONTROL_SEPARATOR( Execution Controls: )
+CLI_CONTROL(cl_uint,        StopOnEnqueueCount,                     UINT_MAX, "If set, CLIntercept will exit the application on achieved enqueue count")
```

Comment:
I'd like to see a few changes here:

1. Please refer to the project as "the Intercept Layer for OpenCL Applications" and not the short-hand "CLIntercept".
2. Please use a `uint64_t` as the datatype for this counter rather than a `cl_uint`.  Some long-running applications make more than 4B enqueues and we don't want to terminate them when the enqueue counter reaches `UINT_MAX`.  If this ends up being problematic for some reason, we could use a `cl_uint`, but ignore the control if it is set to zero, which should satisfy the intended use-cases for the control.  (This would also have some nice symmetry with the `ReportInterval` control.)
3. Please include a full description of the control and use it to generate the controls document rather than editing the controls document manually.
4. Minor/bikeshedding: Should the name of the control be ExitOnEnqueueCount vs. StopOnEnqueueCount to clarify that we're calling `exit()` (and not `abort` or `terminate`, etc.)?

Reply:
Done